test/e2e/deployment.go

```diff
@@ -187,7 +187,12 @@ func stopDeploymentMaybeOverlap(c clientset.Interface, internalClient internalcl
 	reaper, err := kubectl.ReaperFor(extensionsinternal.Kind("Deployment"), internalClient)
 	Expect(err).NotTo(HaveOccurred())
 	timeout := 1 * time.Minute
-	err = reaper.Stop(ns, deployment.Name, timeout, api.NewDeleteOptions(0))
+
+	// Make deployment go away immediately by removing
+	// the orphan finalizer
+	var falseVar = false
+	nonOrphanOption := api.DeleteOptions{OrphanDependents: &falseVar}
```

Comment:
Don't set it here, it's not passed to when the reaper actually deletes the deployment object. Set the DeleteOptions [here](https://github.com/kubernetes/kubernetes/blob/v1.5.2-beta.0/pkg/kubectl/stop.go#L470).